MapboxDirections/MBDirections.swift

```diff
@@ -207,11 +214,12 @@ open class Directions: NSObject {
         return task
     }
     
-    @objc(calculateRoutableMatchWithOptions:completionHandler:)
-    @discardableResult open func routableMatch(_ options: MatchingOptions, completionHandler: @escaping RouteableMatchCompletionHandler) -> URLSessionDataTask {
+    @objc(calculateRoutesMatchingOptions:completionHandler:)
+    @discardableResult open func calculateRoutes(_ options: MatchingOptions, completionHandler: @escaping RouteCompletionHandler) -> URLSessionDataTask {
```

Comment:
Per https://github.com/mapbox/MapboxDirections.swift/pull/236#discussion_r172036785, this method name should be `calculateRoutes(matching:completionHandler:)`, not `calculateRoutes(_:completionHandler:)`.